apps/DensifyPointCloud/DensifyPointCloud.cpp

```diff
@@ -71,6 +72,7 @@ bool Initialize(size_t argc, LPCTSTR* argv)
 		("config-file,c", boost::program_options::value<std::string>(&OPT::strConfigFileName)->default_value(APPNAME _T(".cfg")), "file name containing program options")
 		("archive-type", boost::program_options::value(&OPT::nArchiveType)->default_value(2), "project archive type: 0-text, 1-binary, 2-compressed binary")
 		("process-priority", boost::program_options::value(&OPT::nProcessPriority)->default_value(-1), "process priority (below normal by default)")
+		("export-dmap-only,edo",boost::program_options::value(&OPT::exportDmapOnly)->default_value(true),"only export *.Dmap and *.raw.pfm file")
```

Comment:
Default to false to preserve original behavior.

Are you sure '-edo' option will work?